index.bs

```diff
@@ -2913,41 +2915,47 @@ The sample code for generating and registering a new key follows:
 <pre class="example" highlight="js">
     if (!ScopedCredential) { /* Platform not capable. Handle error. */ }
 
-    var userAccountInformation = {
-        rpDisplayName: "Acme",
-        displayName: "John P. Smith",
-        name: "johnpsmith@example.com",
-        id: "1098237235409872",
-        imageURL: "https://pics.acme.com/00/p/aBjjjpqPb.png"
-    };
+    var scoped = {
+      challenge: new TextEncoded().encode("climb a mountain"),
```

Comment:
I have angst wrt this bogus client-side non-random setting of challenge (see #404) -- perhaps we should change it to (if I have it correct)..
```js
var challenge = Uint8Array.from(window.atob("PGifxAoBwCkWkm4b1CiIl5otCphiIh6MijdjbWFjomA="), c=>c.charCodeAt(0))
``` 
for now as we await #404 to be decided?